<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Word_SentenceexercisetoolClass</class>
 <widget class="QWidget" name="Word_SentenceexercisetoolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word_Sentenceexercisetool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>22</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pb_home">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>回</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_add">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_close">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>叉</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 255, 0)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Word_Sentenceexercisetool.qrc"/>
 </resources>
 <connections/>
</ui>
